docs/ci_example.md

```diff
@@ -0,0 +1,111 @@
+# Example 5: Incorporate fal into a CI/CD pipeline
+You can use fal as part of your CI/CD pipeline. In this example we use [Github Actions](https://github.com/features/actions).
+
+## Environment variables
+Environemt variables need to be provided as [repository secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) These will be different depending on which sources and outputs your project uses. In this example we use [BigQuery](https://cloud.google.com/bigquery/) both as a source and as an output, and so we need to provide BigQuery-specific environment variables:
+
+- `SERVICE_ACCOUNT_KEY`: contents of a `keyfile.json`. For more information see [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+- `GCLOUD_PROJECT`: your Google Cloud project ID.
+- `BQ_DATASET`: name of the BigQuery dataset
+
+If your fal scripts require environment variables, these should also be provided as repository secrets.
+
+## Setup dbt project
+
+### `profiles.yml`
```

Comment:
Is this file to be commited in the repo?

Reply:
yes